changelogs/fragments/add_lxd_virtual_machine.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Add `lxd_virtual_machine`"
```

Comment:
Could you please remove this? New plugins/modules should not have changelog fragments.